### ISSUE TITLE
fieldset missing legend in main job description box on landing page

### ISSUE THREAD

**Issue by dianalam** [USDepartmentofLabor/ableist-language-detector-web#5]
> What It Means
A fieldset does not have a legend.
Why It Matters
A fieldset legend presents a description of the form elements within a fieldset and is especially useful to screen reader users. A legend should be provided when a higher level description is necessary for groups of check boxes, radio buttons, or other form controls.
How to Fix It
If a higher level description is necessary for the user to understand the function or purpose of the controls within the fieldset, provide this description within the <legend>. If this description or grouping is not necessary, the fieldset should probably be removed. Note that the legend is repeated to screen reader users for each form control within the fieldset.
The Algorithm... in English
A fieldset does not have a legend or the legend is empty.
Standards and Guidelines
1.1.1 Non-text Content (Level A)
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)
3.3.2 Labels or Instructions (Level A)
